edxlearndot/apps.py

```diff
@@ -0,0 +1,23 @@
+"""
+Django app configuration
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+import logging
+
+from django.apps import AppConfig
+
+
+log = logging.getLogger(__name__)
+
+
+class LearndotIntegrationConfig(AppConfig):
+    """
+    Configures edxlearndot as a Django app
+    """
+    name = 'edxlearndot'
+    verbose_name = "edX Learndot Integration"
+
+    def ready(self):
+        import edxlearndot.signals  # pylint: disable=unused-variable
```

Comment:
Since we don't want to have to alter the LMS env code for our client, could you please include the [`edxlearndot.settings`](https://github.com/open-craft/edx-learndot/pull/1/files#diff-27d9a09c4f4808c570368ce3984f6a3a) file in this branch, and invoke it here to update the settings from the env?  E.g. 
```python
        from edxlearndot.settings import plugin_settings
        plugin_settings(settings)
```

Will also need to add `from django.conf import settings` above.

Reply:
Done.